src/test/rgw/rgw_multi/tests_ps.py

```diff
@@ -1351,951 +946,178 @@ def test_objcet_timing():
     master_zone.delete_bucket(bucket_name)
 
 
-def test_ps_s3_notification_push_amqp_on_master():
-    """ test pushing amqp s3 notification on master """
+def test_ps_s3_persistent_multiple_gateways():
```

Comment:
all tests starting with "test_ps_s3_persistent..." are running only on master.
should be removed as well

Reply:
ahh sorry, I've added them in new file but have forgotten to remove it from here.